src/defaultConfig.js

```diff
@@ -116,15 +121,22 @@ const config = {
    * call to mermaidAPI.initialize. Calls to mermaidAPI.reinitialize cannot make changes to
    * the `secure` keys in the current currentConfig. This prevents malicious graph directives from
    * overriding a site's default security.
+
+   * **Notes**:
+   *
+   * Default value: ['secure', 'securityLevel', 'startOnLoad', 'maxTextSize']
```

Comment:
I add default value for this one. It did take some time for me to infer that it should be `string[]` before reading souce code to confirm it.

Users should have confidence filling it without checking source code to confirm it's type after declaring the default value.